src/ReferenceList.php

```diff
@@ -24,6 +24,9 @@
  * @author H. Snater < mediawiki@snater.com >
  * @author Thiemo Kreuz
  * @author Bene* < benestar.wikimedia@gmail.com >
+ *
+ * @phpcs:disable MediaWiki.NamingConventions.LowerCamelFunctionsName.FunctionName
```

Comment:
Looks like the mediawiki-codesniffer needs to be fixed for these new magic methods
https://phabricator.wikimedia.org/T253633
Patch already up https://gerrit.wikimedia.org/r/598745

Fine to merge it as it is, and try to remember to remove the disable line when we get to use the new version!
(I could even file a ticket for that)

Reply:
Ok cool, I think this might be a good route!